components/crud-web-apps/jupyter/manifests/base/cluster-role.yaml

```diff
@@ -3,15 +3,6 @@ kind: ClusterRole
 metadata:
   name: cluster-role
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - namespaces
-  verbs:
-  - get
-  - list
```

Comment:
@juliusvonkohout Can you confirm that the jupyter-web-app still functions correctly without "get" and "list" on namespaces?

Reply:
That seems to be the case. I just updated the roles, restarted jupyter-web-app and notebook-controller and was able to start, stop, create, delete jupyterlabs. There  are also no errors in the logs.
[Uploading jupyter-web-app-deployment-7f7f55f9d8-slb8k-jupyter-web-app.log…]()

Reply:
Is there anything more to test or it is ready to be merged?

Reply:
Should be good to go. It would also be a bug from the current code if it couldn't operate without listing namespaces, since it shouldn't need these permissions in the first place.

Note that we might need to re-introduce them though, in a future iteration when we would want the apps to be self-standing. But we need to discuss other items for this effort. But I'm also mentioning this context to keep in mind